packages/@storefront/core/src/core/utils.ts

```diff
@@ -12,6 +12,13 @@ export const WINDOW = () => window;
 
 const ARRAY_TO_DOT_NOTATION_REGEX = /\[(\d+)\]/g;
 
+export const KEYS = {
```

Comment:
Does this table not exist somewhere already?

Reply:
The latter, hoping for a built-in browser object or a library that we already have, but I guess one doesn't exist.